maintainers/maintainer-list.nix

```diff
@@ -2143,6 +2143,12 @@
     githubId = 199180;
     name = "Claes Wallin";
   };
+  cleeyv = {
```

Comment:
Didn't that get catched by ofborg?

Also please mention such fixes in the PR title or motivation so that it is obvious that it was forgotten in the PR.

Reply:
Got it, will make sure to make it clearer next time.
The failure did get catched by ofborg, but probably the person merging didn't see it (or thought the failure was unrelated). Those things happen, very infrequently, but they do happen.
Thank you for taking the time to review this!